BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -652,6 +666,17 @@ public extension AppProcessor {
     }
 }
 
+// MARK: - AccountTokenProviderDelegate
+
+extension AppProcessor: AccountTokenProviderDelegate {
+    func onRefreshTokenError(error: any Error) async throws {
+        if let identityTokenError = error as? IdentityTokenRefreshRequestError,
+           identityTokenError == .invalidGrant {
```

Comment:
🎨 This could be simplified slightly:
```suggestion
        if case IdentityTokenRefreshRequestError.invalidGrant = error {
```